live/tests/unit/models/assessment-test.js

```diff
@@ -45,18 +46,82 @@ describe('Unit | Model | Assessment', function() {
         expect(hasCheckpoints).to.be.false;
       });
     });
-
   });
 
-  describe('on ready event', function() {
-    it('should instanciate an assessment progression behavior', function() {
-      run(() => {
-        const store = this.store();
-        // when
-        const assessment = store.createRecord('assessment');
-        // then
-        expect(assessment._progressionBehavior).to.be.ok;
+  describe('@answersSinceLastCheckpoints', function() {
+
+    function newAnswers(store, nbAnswers) {
+      return run(() => {
+        return _.times(nbAnswers, () => store.createRecord('answer'));
       });
+    }
+
+    it('should return an empty array when no answers has been given', function() {
+      // given
+      const assessment = this.subject();
+      assessment.set('answers', []);
```

Comment:
si je ne m'abuse, tu peux faire `const assessment = this.subject({ answers: [] });